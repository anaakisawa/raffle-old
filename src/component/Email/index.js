import React, { Component } from 'react';
import Header from '../Header';
import Input from '../Input';
import Button from '../Button';

class Email extends Component {
  render() {
    return(
      <div className="Email">
        <Header text="Qual o seu email?"/>
        <Input/>
        <Button text="Continuar"/>
      </div>
    );
  }
}

export default Email;
