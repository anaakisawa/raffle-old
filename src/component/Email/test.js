
import React from 'react';
import Email from './index';
import renderer from 'react-test-renderer';

describe('Component - Email', () => {
  test('renders an email form', () => {
    const component = renderer.create(
      <Email/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});


/* */
