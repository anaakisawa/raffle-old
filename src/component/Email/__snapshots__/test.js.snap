// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Email renders an email form 1`] = `
<div
  className="Email"
>
  <div
    className="Header"
  >
    Qual o seu email?
  </div>
  <div
    className="Input"
  >
    <input
      type="text"
    />
  </div>
  <button
    className="Button"
  >
    Continuar
  </button>
</div>
`;
